python/foxglove-sdk/src/schemas.rs

```diff
@@ -0,0 +1,1921 @@
+//! Definitions for well-known Foxglove schemas
+//! Generated by https://github.com/foxglove/schemas
+#![allow(clippy::too_many_arguments)]
+#![allow(clippy::enum_variant_names)]
+#![allow(non_snake_case)]
+use pyo3::prelude::*;
+
+/// An enumeration indicating how input points should be interpreted to create lines
+#[pyclass(eq, eq_int)]
+#[derive(PartialEq, Clone)]
+pub(crate) enum LinePrimitiveLineType {
```

Comment:
thoughts on generating each of these as separate files?

if not, at least alphabetically sorting them would be a good idea

Reply:
oh I guess maybe they are alpha-sorted after the enums? I can't tell.